ChaosMod/Memory/Hooks/HandleToEntityStructHook.cpp

```diff
@@ -10,13 +9,16 @@ __int64 _HK_HandleToEntityStruct(Entity entity)
 	{
 		return 0;
 	}
+	Entity vehToContinue = entity;
 
-	if (entity == m_origHandle)
+	while (vehicleMap.count(vehToContinue) > 0)
```

Comment:
Shouldn't this be an `if` instead of `while`? (Also an `empty` function exists already which does the same check)

Reply:
nah, im resolving the id here, as it can be multiple layers.
id 1 orig -> id 2
id 2 -> id 3
id 3 -> id 4

asking for id 1 would therefore be resolved to id4 :)

Reply:
Ah I see. Also disregard my latter statement, I didn't see the `vehToContinue` passed to `count`.